hotsos/defs/scenarios/openstack/cinder/bugs/lp2085851.yaml

```diff
@@ -0,0 +1,97 @@
+      - apt:
+          cinder-api:
+            # caracal
+            - eq: '2:24.0.0-0ubuntu1.2'
+            # bobcat
+            - eq: '2:23.0.0-0ubuntu1.4~cloud0'
+            # antelope
+            - min: '2:22.1.1-0ubuntu1.3~cloud0'
+              max: '2:22.1.1-0ubuntu1.3~cloud1'
+            # yoga
+            - eq: '2:20.3.1-0ubuntu1.4'
+            # ussuri
+            - eq: '2:16.4.2-0ubuntu2.8'
+      - apt:
+          cinder-backup:
+            # caracal
+            - eq: '2:24.0.0-0ubuntu1.2'
+            # bobcat
+            - eq: '2:23.0.0-0ubuntu1.4~cloud0'
+            # antelope
+            - min: '2:22.1.1-0ubuntu1.3~cloud0'
+              max: '2:22.1.1-0ubuntu1.3~cloud1'
+            # yoga
+            - eq: '2:20.3.1-0ubuntu1.4'
+            # ussuri
+            - eq: '2:16.4.2-0ubuntu2.8'
+      - apt:
+          cinder-common:
+            # caracal
+            - eq: '2:24.0.0-0ubuntu1.2'
+            # bobcat
+            - eq: '2:23.0.0-0ubuntu1.4~cloud0'
+            # antelope
+            - min: '2:22.1.1-0ubuntu1.3~cloud0'
+              max: '2:22.1.1-0ubuntu1.3~cloud1'
+            # yoga
+            - eq: '2:20.3.1-0ubuntu1.4'
+            # ussuri
+            - eq: '2:16.4.2-0ubuntu2.8'
+      - apt:
+          cinder-scheduler:
+            # caracal
+            - eq: '2:24.0.0-0ubuntu1.2'
+            # bobcat
+            - eq: '2:23.0.0-0ubuntu1.4~cloud0'
+            # antelope
+            - min: '2:22.1.1-0ubuntu1.3~cloud0'
+              max: '2:22.1.1-0ubuntu1.3~cloud1'
+            # yoga
+            - eq: '2:20.3.1-0ubuntu1.4'
+            # ussuri
+            - eq: '2:16.4.2-0ubuntu2.8'
+      - apt:
+          cinder-volume:
+            # caracal
+            - eq: '2:24.0.0-0ubuntu1.2'
+            # bobcat
+            - eq: '2:23.0.0-0ubuntu1.4~cloud0'
+            # antelope
+            - min: '2:22.1.1-0ubuntu1.3~cloud0'
+              max: '2:22.1.1-0ubuntu1.3~cloud1'
+            # yoga
+            - eq: '2:20.3.1-0ubuntu1.4'
+            # ussuri
+            - eq: '2:16.4.2-0ubuntu2.8'
+      - apt:
+          python3-cinder:
+            # caracal
+            - eq: '2:24.0.0-0ubuntu1.2'
+            # bobcat
+            - eq: '2:23.0.0-0ubuntu1.4~cloud0'
+            # antelope
+            - min: '2:22.1.1-0ubuntu1.3~cloud0'
+              max: '2:22.1.1-0ubuntu1.3~cloud1'
+            # yoga
+            - eq: '2:20.3.1-0ubuntu1.4'
+            # ussuri
+            - eq: '2:16.4.2-0ubuntu2.8'
+conclusions:
+  pkg_is_vulnerable:
+    decision:
+      - pkg_with_regression
+    raises:
+      type: LaunchpadBug
+      bug-id: 2085851
+      message: >-
+        The version of Openstack Cinder ({version}) running on this host is
+        impacted by regression LP2085851. There is a fix available in the Ubuntu
+        archives and upgrading is recommended. Please check
+        https://bugs.launchpad.net/ubuntu/+source/cinder/+bug/2085851 for more information.
```

Comment:
Maybe skip printing the bug url in the message since it will already be printed by hotsos

Reply:
Done